processing/src/main/java/io/druid/segment/filter/ExtractionFilter.java

```diff
@@ -65,12 +66,14 @@ public ExtractionFilter(
     return filters;
   }
 
+  private static final WrappedImmutableConciseBitmap ZERO_LENGTH_SET = new WrappedImmutableConciseBitmap(new ImmutableConciseSet());
```

Comment:
This is similar to what is done in `io.druid.segment.data.ConciseBitmapSerdeFactory.ImmutableConciseSetObjectStrategy#fromByteBuffer`

I am curious now if this PR will work with roaring dimensions.